aggregate/aggregate

```diff
@@ -188,6 +198,91 @@ def main():
         "Other_resources"
     ]]
 
+    # Parse pdbminer complexes output:
+    for index, row in pdbminer_c_df.iterrows():
+        structure_id = row["structure_id"]
+        complex_details = row["complex_details"]
+
+        # Remove asterisks from structure IDs:
+        normalized_structures = final_df["Structure"].apply(lambda x: ";".join(s.replace("*", "").replace("**", "") for s in x.split(";")))
+
+        # Check if any structure_id is not in final_df:
+        if not any(structure_id in existing_structure.split(";") for existing_structure in normalized_structures):
+            split_details = re.split(r"[,\s;]+", complex_details)
+
+            #Retrieve the upacs of that structure:
+            uniprot_ids = [part for part in split_details if re.match(r"[OPQ][0-9][A-Z0-9]{3}[0-9]|[A-NR-Z][0-9]([A-Z0-9]{3,5})", part)]
+
+            for uniprot_id in uniprot_ids:
+                # exclude the target protein:
+                if uniprot_id != final_df['Target_Uniprot_AC'].iloc[0]:
+                    
+                    # Check if interactor is already in final_df:
+                    matching_row = final_df[final_df["Interactor_UniProt_AC"] == uniprot_id]
+                    
+                    if not matching_row.empty:
```

Comment:
can we have

```
if matching_row.empty:
    ....
else:
    ....
```

this is more straightforward to read and understand, and has the same outcome